src/selectors/isPending.ts

```diff
@@ -3,7 +3,7 @@ import Thought from '../@types/Thought'
 import { getAllChildrenAsThoughts } from '../selectors/getChildren'
 
 /** Returns true if the context has not been loaded from the remote yet, or if the children are not loaded. */
-const isPending = (state: State, thought: Thought | null): boolean => {
+const isPending = (state: State, thought: Thought | null | undefined): boolean => {
```

Comment:
this is the "controversial" part I guess
explained in the PR itself

Reply:
I think `Thought | undefined` should cover it. I'm not sure why it was using `Thought | null`. Thanks!